pkg/inventory/db.go

```diff
@@ -52,9 +56,15 @@ var (
 type DB struct {
 	instance *cloudprovider.CloudInstance
 
-	mu         sync.RWMutex
-	podStore   map[int]string    // key: netnsid path value: Pod namespace/name
-	podNsStore map[string]string // key pod value: netns path
+	mu sync.RWMutex
+	// netNsForPod gives the network namespace for a pod, indexed by the pods
+	// "namespaced/name".
+	netNsForPod map[string]string
```

Comment:
Nit: This name seems almost like a function declaration, not a map. Something similar to what was before like netNsStore seems preferable imo.